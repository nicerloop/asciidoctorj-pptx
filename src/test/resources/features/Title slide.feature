Feature: Title slide

    Background:
        Given I have an empty temporary asciidoc document

    Scenario: Title
        Given the asciidoc document has a title
        When I convert the asciidoc document to an Office Open XML slideshow
        Then the slideshow has 1 slide
        And the slide looks like expected

    Scenario: Title and subtitle
        Given the asciidoc document has a title and a subtitle
        When I convert the asciidoc document to an Office Open XML slideshow
        Then the slideshow has 1 slide
        And the slide looks like expected

    # Scenario: Title and author
    #     Given the asciidoc document has a title
    #     And the asciidoc document has an author
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected

    # Scenario: Title and date
    #     Given the asciidoc document has a title
    #     And the asciidoc document has a version and a date
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected

    # Scenario: Title, subtitle, author and date
    #     Given the asciidoc document has a title and a subtitle
    #     And the asciidoc document has an author and a date
    #     When I convert the asciidoc document to an Office Open XML slideshow
    #     Then the slideshow has 1 slide
    #     And the slide looks like expected
