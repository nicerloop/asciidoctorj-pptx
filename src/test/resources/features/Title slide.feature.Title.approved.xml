<?xml version="1.0" encoding="UTF-8"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="text/ecmascript" contentStyleType="text/css" preserveAspectRatio="xMidYMid meet" style="color-interpolation:auto; color-rendering:auto; fill:black; fill-opacity:1; font-family:'Dialog'; font-size:12px; font-style:normal; font-weight:normal; image-rendering:auto; shape-rendering:auto; stroke:black; stroke-dasharray:none; stroke-dashoffset:0; stroke-linecap:square; stroke-linejoin:miter; stroke-miterlimit:10; stroke-opacity:1; stroke-width:1; text-rendering:auto;" version="1.0" zoomAndPan="magnify">
  <defs id="genericDefs"/>
  <g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:white; image-rendering:optimizeQuality; stroke:white; text-rendering:optimizeLegibility;">
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-opacity:0; text-rendering:optimizeLegibility;">
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <rect height="540" style="stroke:none;" width="720" x="0" y="0"/>
      <path d="M54 167.75 L666 167.75 L666 283.5 L54 283.5 Z" style="stroke:none; stroke-linecap:butt; stroke-linejoin:round; stroke-width:0.25;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; image-rendering:optimizeQuality; stroke-linecap:butt; stroke-linejoin:round; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M341.9912 210.4062 Q341.9912 210.8281 341.9443 211.125 Q341.8975 211.4219 341.8037 211.6094 Q341.71 211.7812 341.5693 211.875 Q341.4287 211.9531 341.2568 211.9531 L333.4287 211.9531 L333.4287 236.0625 Q333.4287 236.2344 333.335 236.375 Q333.2568 236.5 333.0381 236.5781 Q332.835 236.6406 332.4756 236.6875 Q332.1318 236.75 331.585 236.75 Q331.0693 236.75 330.6943 236.6875 Q330.335 236.6406 330.1162 236.5781 Q329.9131 236.5 329.8193 236.375 Q329.7412 236.2344 329.7412 236.0625 L329.7412 211.9531 L321.9131 211.9531 Q321.7412 211.9531 321.6006 211.875 Q321.4756 211.7812 321.3818 211.6094 Q321.3037 211.4219 321.2412 211.125 Q321.1943 210.8281 321.1943 210.4062 Q321.1943 210 321.2412 209.6875 Q321.3037 209.375 321.3818 209.1875 Q321.4756 209 321.6006 208.9219 Q321.7412 208.8281 321.9131 208.8281 L341.2568 208.8281 Q341.4287 208.8281 341.5693 208.9219 Q341.71 209 341.8037 209.1875 Q341.8975 209.375 341.9443 209.6875 Q341.9912 210 341.9912 210.4062 ZM349.1357 236.0938 Q349.1357 236.2656 349.042 236.3906 Q348.9639 236.5 348.7764 236.5938 Q348.5889 236.6719 348.2451 236.7031 Q347.9014 236.75 347.3545 236.75 Q346.8389 236.75 346.4951 236.7031 Q346.1514 236.6719 345.9482 236.5938 Q345.7451 236.5 345.667 236.3906 Q345.5889 236.2656 345.5889 236.0938 L345.5889 216.75 Q345.5889 216.5938 345.667 216.4688 Q345.7451 216.3438 345.9482 216.2656 Q346.1514 216.1719 346.4951 216.125 Q346.8389 216.0781 347.3545 216.0781 Q347.9014 216.0781 348.2451 216.125 Q348.5889 216.1719 348.7764 216.2656 Q348.9639 216.3438 349.042 216.4688 Q349.1357 216.5938 349.1357 216.75 L349.1357 236.0938 ZM349.542 210.2188 Q349.542 211.4688 349.0732 211.9219 Q348.6045 212.375 347.3389 212.375 Q346.0889 212.375 345.6201 211.9375 Q345.167 211.4844 345.167 210.2656 Q345.167 209.0156 345.6357 208.5625 Q346.1045 208.1094 347.3857 208.1094 Q348.6201 208.1094 349.0732 208.5625 Q349.542 209 349.542 210.2188 ZM365.749 234.6719 Q365.749 235.2969 365.6553 235.6562 Q365.5771 236.0156 365.4053 236.1875 Q365.2334 236.3594 364.8896 236.5156 Q364.5459 236.6719 364.1084 236.7656 Q363.6709 236.8594 363.1709 236.9219 Q362.6709 236.9844 362.1865 236.9844 Q360.6709 236.9844 359.5928 236.5938 Q358.5303 236.1875 357.8428 235.3906 Q357.1553 234.5781 356.8428 233.3438 Q356.5303 232.1094 356.5303 230.4375 L356.5303 219.1406 L353.8271 219.1406 Q353.499 219.1406 353.2959 218.7969 Q353.1084 218.4531 353.1084 217.6719 Q353.1084 217.2656 353.1553 216.9844 Q353.2178 216.7031 353.2959 216.5312 Q353.3896 216.3438 353.5303 216.2656 Q353.6709 216.1875 353.8428 216.1875 L356.5303 216.1875 L356.5303 211.5938 Q356.5303 211.4375 356.6084 211.3125 Q356.6865 211.1875 356.8896 211.0938 Q357.0928 211 357.4365 210.9531 Q357.7803 210.9062 358.2959 210.9062 Q358.8271 210.9062 359.1709 210.9531 Q359.5146 211 359.7021 211.0938 Q359.9053 211.1875 359.9834 211.3125 Q360.0771 211.4375 360.0771 211.5938 L360.0771 216.1875 L365.0459 216.1875 Q365.2178 216.1875 365.3428 216.2656 Q365.4678 216.3438 365.5615 216.5312 Q365.6553 216.7031 365.7021 216.9844 Q365.749 217.2656 365.749 217.6719 Q365.749 218.4531 365.5459 218.7969 Q365.3584 219.1406 365.0459 219.1406 L360.0771 219.1406 L360.0771 229.9219 Q360.0771 231.9219 360.6709 232.9375 Q361.2646 233.9531 362.7803 233.9531 Q363.2803 233.9531 363.6553 233.8594 Q364.0459 233.7656 364.3428 233.6562 Q364.6553 233.5469 364.8584 233.4531 Q365.0771 233.3594 365.249 233.3594 Q365.3584 233.3594 365.4521 233.4219 Q365.5615 233.4688 365.6084 233.625 Q365.6553 233.7656 365.7021 234.0312 Q365.749 234.2812 365.749 234.6719 ZM373.9717 236.0938 Q373.9717 236.2656 373.8779 236.3906 Q373.7998 236.5 373.6123 236.5938 Q373.4248 236.6719 373.0811 236.7031 Q372.7373 236.75 372.1904 236.75 Q371.6748 236.75 371.3311 236.7031 Q370.9873 236.6719 370.7842 236.5938 Q370.5811 236.5 370.5029 236.3906 Q370.4248 236.2656 370.4248 236.0938 L370.4248 207.3906 Q370.4248 207.2188 370.5029 207.0938 Q370.5811 206.9531 370.7842 206.875 Q370.9873 206.7812 371.3311 206.75 Q371.6748 206.7031 372.1904 206.7031 Q372.7373 206.7031 373.0811 206.75 Q373.4248 206.7812 373.6123 206.875 Q373.7998 206.9531 373.8779 207.0938 Q373.9717 207.2188 373.9717 207.3906 L373.9717 236.0938 ZM397.0225 225.5781 Q397.0225 226.4219 396.6006 226.7812 Q396.1943 227.125 395.6475 227.125 L382.9756 227.125 Q382.9756 228.7344 383.2881 230.0312 Q383.6162 231.3125 384.3662 232.25 Q385.1318 233.1719 386.335 233.6719 Q387.5381 234.1562 389.2725 234.1562 Q390.6475 234.1562 391.7256 233.9375 Q392.8037 233.7031 393.585 233.4219 Q394.3662 233.1406 394.8662 232.9219 Q395.3818 232.6875 395.6318 232.6875 Q395.7881 232.6875 395.8975 232.7656 Q396.0225 232.8438 396.085 233 Q396.1475 233.1406 396.1787 233.4219 Q396.21 233.6875 396.21 234.0625 Q396.21 234.3438 396.1787 234.5469 Q396.1631 234.75 396.1318 234.9219 Q396.1006 235.0781 396.0225 235.2031 Q395.96 235.3281 395.835 235.4531 Q395.7256 235.5781 395.1318 235.8438 Q394.5537 236.1094 393.6318 236.375 Q392.71 236.625 391.4912 236.8281 Q390.2881 237.0312 388.9131 237.0312 Q386.5225 237.0312 384.7256 236.375 Q382.9287 235.7031 381.71 234.3906 Q380.4912 233.0781 379.8662 231.1094 Q379.2412 229.125 379.2412 226.5 Q379.2412 224.0156 379.8818 222.0312 Q380.5225 220.0312 381.7412 218.6562 Q382.96 217.2656 384.6787 216.5312 Q386.3975 215.7812 388.5225 215.7812 Q390.8037 215.7812 392.3975 216.5156 Q394.0068 217.25 395.0381 218.4844 Q396.0693 219.7188 396.5381 221.3906 Q397.0225 223.0469 397.0225 224.9375 L397.0225 225.5781 ZM393.46 224.5312 Q393.5225 221.7344 392.2256 220.1562 Q390.9287 218.5625 388.3662 218.5625 Q387.0537 218.5625 386.0693 219.0625 Q385.085 219.5469 384.4131 220.3594 Q383.7568 221.1719 383.3818 222.2656 Q383.0225 223.3438 382.9756 224.5312 L393.46 224.5312 Z" style="stroke:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M54 167.75 L666 167.75 L666 283.5 L54 283.5 Z" style="fill:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M108 306 L612 306 L612 444 L108 444 Z" style="stroke:none;"/>
    </g>
    <g style="color-interpolation:linearRGB; color-rendering:optimizeQuality; fill:rgb(255,255,255); fill-opacity:0; image-rendering:optimizeQuality; stroke:rgb(255,255,255); stroke-linecap:butt; stroke-linejoin:round; stroke-opacity:0; stroke-width:0.25; text-rendering:optimizeLegibility;">
      <path d="M108 306 L612 306 L612 444 L108 444 Z" style="fill:none;"/>
    </g>
  </g>
</svg>
